arangod/Transaction/Options.h

```diff
@@ -87,6 +87,8 @@ struct Options {
   /// created.
   bool allowDirtyReads = false;
 
+  bool isEarlyReadOnly = true;
```

Comment:
What does this flag do? Under what circumstances can it be used? At what time is its value consided valid? Code comments would help here...

Reply:
I would prefer to **not** have `ìsEarlyReadOnly` inside the transaction's options, because it is not something that an end user should set from the outside. And the flag only seems to be used inside `transaction::Manager`, so can we remove it again from the options?